lib/note-content-editor.tsx

```diff
@@ -169,16 +203,17 @@ class NoteContentEditor extends Component<Props> {
       this.state.editor === 'full' &&
       prevProps.searchQuery !== this.props.searchQuery
     ) {
+      this.setState({ selectedSearchMatchIndex: null });
```

Comment:
noting so it doesn't get lost in the changing code: this `setState()` shouldn't be here; it's more appropriate inside of `getDerivedStateFromProps()`

Reply:
done